action-server/covidflow/setup_structlog.py

```diff
@@ -36,9 +37,12 @@ def setup_structlog(
         default = "pretty" if sys.stdout.isatty() else "json"
         log_style = environ.get("log_style", default)
 
-    if not logging.root.handlers:
-        handler = logging.StreamHandler()
-        logging.root.handlers.append(handler)
+    handler = logging.StreamHandler()
+    handler.setFormatter(CustomJsonFormatter())
```

Comment:
ça devrait seulement être fait dans le cas où log_style = json

Reply:
Moved it back to the log_style condition